### ISSUE TITLE
[Bug]: User friendly error message: Error: Video file D:\stable-diffusion-webui has format 'd:\stable-diffusion-webui', which is not supported. Supported formats are: ['mov', 'mpeg', 'mp4', 'm4v', 'avi', 'mpg', 'webm']. Please, check your schedules/ init values.

### ISSUE THREAD

**Issue by Gcode808** [Mikubill/sd-webui-controlnet#1926]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

It was working fine yesterday with the exact same settings. Now I get this error:

User friendly error message:
Error: Video file D:\stable-diffusion-webui has format 'd:\stable-diffusion-webui', which is not supported. Supported formats are: ['mov', 'mpeg', 'mp4', 'm4v', 'avi', 'mpg', 'webm']. Please, check your schedules/ init values.

### Steps to reproduce the problem

1. Go to .... 
2. Press ....
3. ...

### What should have happened?

it should have generated the deforum hybrid video.

### Commit where the problem happens

webui: 
controlnet: 

in deforum when using controlnet

### What browsers do you use to access the UI ?

Google Chrome

### Command Line Arguments

```Shell
--medvram --xformers --ckpt-dir 'D:\stable-diffusion-webui\models\Stable-diffusion' --theme dark
```

### List of enabled extensions

controlnet and deforum only

### Console logs

```Shell
started run_deforum
times_to_run: 1
Deforum extension for auto1111 webui
Git commit: 19023b1d
Starting job task(wuh2jpjw2cn6ozw)-0...
Video to extract: D:\test-deforum\1024x576\1024x576.mp4
Extracting video (1 every 1) frames to D:\stable-diffusion-webui\outputs\img2img-images\AICz\inputframes...
Trying to extract frames from video with input FPS of 30.0. Please wait patiently.
Extracted 261 frames from video in 0.98 seconds!
Using 261 input frames from D:\stable-diffusion-webui\outputs\img2img-images\AICz\inputframes...
Using init_image from video: D:\stable-diffusion-webui\outputs\img2img-images\AICz\inputframes\1024x576000000000.jpg
Unpacking ControlNet 1 base video
Exporting Video Frames to D:\stable-diffusion-webui\outputs\img2img-images\AICz\controlnet_1_inputframes...
Trying to extract frames from video with input FPS of 30.0. Please wait patiently.
Extracted 261 frames from video in 0.96 seconds!
Loading 261 input frames from D:\stable-diffusion-webui\outputs\img2img-images\AICz\controlnet_1_inputframes and saving video frames to D:\stable-diffusion-webui\outputs\img2img-images\AICz
ControlNet 1 base video unpacked!
Unpacking ControlNet 1 video mask
Exporting Video Frames to D:\stable-diffusion-webui\outputs\img2img-images\AICz\controlnet_1_maskframes...

*START OF TRACEBACK*
Traceback (most recent call last):
  File "D:\stable-diffusion-webui\extensions\deforum-for-automatic1111-webui\scripts\deforum_helpers\run_deforum.py", line 110, in run_deforum
    render_animation(args, anim_args, video_args, parseq_args, loop_args, controlnet_args, root)
  File "D:\stable-diffusion-webui\extensions\deforum-for-automatic1111-webui\scripts\deforum_helpers\render.py", line 81, in render_animation
    unpack_controlnet_vids(args, anim_args, controlnet_args)
  File "D:\stable-diffusion-webui\extensions\deforum-for-automatic1111-webui\scripts\deforum_helpers\deforum_controlnet.py", line 343, in unpack_controlnet_vids
    process_controlnet_input_frames(args, anim_args, controlnet_args, None, mask_path, 'maskframes', i)
  File "D:\stable-diffusion-webui\extensions\deforum-for-automatic1111-webui\scripts\deforum_helpers\deforum_controlnet.py", line 313, in process_controlnet_input_frames
    vid2frames(
  File "D:\stable-diffusion-webui\extensions\deforum-for-automatic1111-webui\scripts\deforum_helpers\video_audio_utilities.py", line 87, in vid2frames
    if is_vid_path_valid(video_path):
  File "D:\stable-diffusion-webui\extensions\deforum-for-automatic1111-webui\scripts\deforum_helpers\video_audio_utilities.py", line 171, in is_vid_path_valid
    raise ValueError(f"Video file {video_path} has format '{extension}', which is not supported. Supported formats are: {file_formats}")
ValueError: Video file D:\stable-diffusion-webui has format 'd:\stable-diffusion-webui', which is not supported. Supported formats are: ['mov', 'mpeg', 'mp4', 'm4v', 'avi', 'mpg', 'webm']
*END OF TRACEBACK*

User friendly error message:
Error: Video file D:\stable-diffusion-webui has format 'd:\stable-diffusion-webui', which is not supported. Supported formats are: ['mov', 'mpeg', 'mp4', 'm4v', 'avi', 'mpg', 'webm']. Please, check your schedules/ init values.
```

**Comment by calvinsunx** [comment]
same problem!!

**Comment by RuthlessParadise** [comment]
Same exact problem. Any assistance appreciated

**Comment by huchenlei** [comment]
Please file an issue in deforum repo as the stacktrace does not involve code in this repo.

**Comment by hoaphan2023** [comment]
same problem

**Comment by nawnawdqd** [comment]
Same, need help plz

**Comment by qiyou** [comment]
i'm experiencing the same issue for the past 3 hours. whether using direct Stable Diffusion or A1111 Web UI Autoinstaller

**Comment by nbalive2k** [comment]
same here
But you need to post on deforum repo

**Comment by qiyou** [comment]
i just checke,d if you don't enable ControlNet, Deforum is able to generate. 

github's S4f3ty_Marc at Discord said, "you'll have roll back to a saved config or wait for a fix"

github's RifRaf rerolling steps,
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/2118337/6df80cc0-4dfa-4f33-a40f-93c3a474de9d)

my screenshot below,
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/2118337/9b38e75c-c323-408c-a13e-10ffae1238c4)
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/2118337/e2ba4200-cd70-49f7-974b-8d0f3840abb9)

After I followed RifRaf's steps, I opened Stable Diffusion, made a back up ( for the first time lol ) , turned on ControlNet, and Deforum was able to generate again. 

Best of luck!
